NSrtm.Core/Pgm/Grid/IPgmGrid.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NSrtm.Core.Pgm.Grid
+{
+    public interface IPgmGrid
```

Comment:
According to this article (https://en.wikipedia.org/wiki/Lattice_graph ) i will use grid graph.